cluster/engine.go

```diff
@@ -116,6 +116,7 @@ type Engine struct {
 func NewEngine(addr string, overcommitRatio float64, opts *EngineOpts) *Engine {
 	e := &Engine{
 		Addr:            addr,
+		Name:            "(unknown)",
```

Comment:
I would like to have this added only in info.
It may have some side effects here.